internal/web/templates/pages/settings.html

```diff
@@ -28,6 +28,38 @@ <h2 class="text-reset">Package Repositories</h2>
           {{ end }}
         </div>
       </div>
+      <div class="mt-2">
+        <h2 class="text-reset">Danger Zone</h2>
+        <div class="alert alert-warning" role="alert">
+          <div class="form-switch mb-2">
+            <input
+              class="form-check-input"
+              type="checkbox"
+              role="switch"
+              name="enableAdvancedOptions"
+              id="enableAdvancedOptions" />
+            <label class="form-check-label mx-1 fw-semibold" for="enableAdvancedOptions">Advanced Options</label>
+          </div>
+          <i class="bi bi-exclamation-triangle-fill text-warning me-1"></i>
+          It is not recommended to use these except for testing / maintaining a Glasskube package. These should only
+          ever be used in dev/test environments, and never in production. <b>Use at your own risk!</b>
```

Comment:
I think there is missing a short description about what the advanced options do.

Also, `<strong>` should be preferred over `b`.